src/virtual-source-buffer.js

```diff
@@ -457,11 +469,14 @@ export default class VirtualSourceBuffer extends videojs.EventTarget {
     if (this.videoBuffer_) {
       sortedSegments.video.segments.unshift(sortedSegments.video.initSegment);
       sortedSegments.video.bytes += sortedSegments.video.initSegment.byteLength;
+      this.pendingUpdateEnds_ += 1;
```

Comment:
This line (and removing corresponding audio one below) could be placed in the `try` block of `concatAndAppendSegments_` just before the call to `appendBuffer` on the native buffer